CHANGELOG.md

```diff
@@ -4,12 +4,17 @@
 
 ### Routing
 
-* Added `Router.finishRouting()` method to finish routing session without dismissing related UI and logic components.([#3880](https://github.com/mapbox/mapbox-navigation-ios/pull/3880))
+* Added `Router.finishRouting()` method to finish routing session without dismissing related UI and logic components. ([#3880](https://github.com/mapbox/mapbox-navigation-ios/pull/3880))
 
 ### Map
 
 * Added the `layerPosition` parameter to the `NavigationMapView.show(_:layerPosition:legIndex:)` method for controlling the position of the main route layer while presenting routes. ([#3897](https://github.com/mapbox/mapbox-navigation-ios/pull/3897))
 
+### CarPlay
+
+* Fixed an issue where shields would become invisible when users drive through the tunnel. ([#3882](https://github.com/mapbox/mapbox-navigation-ios/pull/3882))
+* Added the ability to change map style depending on the option that was set in `Always show dark maps`. ([#3882](https://github.com/mapbox/mapbox-navigation-ios/pull/3882))
```

Comment:
This is under the v2.6.0 section, but the milestone is v2.5.0 – which release should it land in?

Reply:
Definitely `v2.6.0`, too late to land it in scope of `v2.5.0` because of potential issues. We'll have more time to test in scope of `v2.6.0`, changed milestone.